tests/test_lag_2.py

```diff
@@ -0,0 +1,128 @@
+import pytest
+import time
+import logging
+from ptf_runner import ptf_runner
+from platform_fixtures import conn_graph_facts
+
+@pytest.fixture(scope="module")
+def common_setup_teardown(duthost, ptfhost, conn_graph_facts):
+    # TODO: get the testbed_type
```

Comment:
I saw there are some test cases improted ptf & ptf.testutils & ... and use BaseTest to get the testbed_type.
I am not very clear on the implementation of ptf package, seems like the source code is not in this repo.